front/src/applications/stdcm/components/StdcmResults/SimulationReportSheet.tsx

```diff
@@ -40,8 +43,8 @@ const SimulationReportSheet = ({
   stdcmData,
   consist,
   simulationReportSheetNumber,
-  mapCanvas,
   operationalPointsList,
+  username,
```

Comment:
userName would be better
```suggestion
  userName,
```